### ISSUE TITLE
Possible XSS

### ISSUE THREAD

**Issue by omni5cience** [cdnjs/new-website#33]
Content from a library's `"description"` field in `package.json` is not properly escaped on the [cdnjs.com](https://cdnjs.com) search results page.

For example:
1.  Go to [cdnjs.com](http://cdnjs.com#search-box)
2.  Search for "bootstrap-select"
3.  Observe the rendered HTML (`<select>`) from the library's [package.json](https://github.com/cdnjs/cdnjs/blob/897309f6a5d451abf9d76c379747284cc9222a83/ajax/libs/bootstrap-select/package.json#L5)

It looks like [this simple HTML-string building](https://github.com/cdnjs/new-website/blob/b54c8fd60777ae90a18d2234edfcac88b9e8ecbf/public/js/main.js#L346) is the culprit. My inclination would be to swap in a simple template library, rather than manually escape all the user-data inputs. Another equally valid solution would be to escape the data in the `package.json` when you add/update the library.

**Comment by PeterDaveHello** [comment]
mention @thomasdavis  @drewfreyling  @ryankirkman

**Comment by thomasdavis** [comment]
Thanks for this!

**Comment by crissdev** [comment]
Found it too while searching for bootstrap. Might be a good idea to fix this since we ca create accounts on the website.

**Comment by jonathanKingston** [comment]
@PeterDaveHello can this commit be merged?

**Comment by redox** [comment]
Unfortunately, the `_highlightResult.description.value` contains some `<em></em>` tags to show where the match occurred. I'm used to write something like:

```js
_.escape(hit._highlightResult.description && hit._highlightResult.description.value).replace(/&lt;(\/?)em&gt;/g, '<$1em>')
```

What do you think guys ?

**Comment by jonathanKingston** [comment]
So the search API is actually broken or the packages.min.json that populates it?
(Sorry mostly trying to figure out how packages.min.json gets created)

**Comment by jonathanKingston** [comment]
Ah replicated the issue. I would change the algolia search to change the `<em>` to use `highlightPreTag=**&highlightPostTag=**` then filter through markdown the results.

https://www.algolia.com/doc/curl#Settings

**Comment by PeterDaveHello** [comment]
Should be fixed in #65
